js/StatusBar.js

```diff
@@ -10,7 +10,7 @@
 
 'use strict';
 
-import React from 'react'
+import * as React from 'react'
```

Comment:
Flow will not infer types with esm imports without * as. Now we are using `React.Node` in the file